world_boss/app/api.py

```diff
@@ -28,8 +35,12 @@
 
 
 @api.route("/ping")
-def pong() -> str:
-    return "pong"
+def pong() -> Response:
+    try:
+        db.session.execute("select 1")
+        return jsonify(message="pong")
+    except Exception:
+        abort(503)
```

Comment:
I think that it would be helpful if we return 503 in json format (as unified form) too.